packages/base/src/util/InvisibleMessage.js

```diff
@@ -0,0 +1,48 @@
+import InvisibleMessageMode from "../types/InvisibleMessageMode.js";
+import getSingletonElementInstance from "./getSingletonElementInstance.js";
+
+const styles = `position: absolute;
+	clip: rect(1px,1px,1px,1px);
+	user-select: none;
+	left: -1000px;
+	top: -1000px;
+	pointer-events: none;`;
+
+const politeSpan = document.createElement("span");
+const assertiveSpan = document.createElement("span");
+
+politeSpan.classList.add("ui5-invisiblemessage-polite");
+assertiveSpan.classList.add("ui5-invisiblemessage-assertive");
+
+politeSpan.setAttribute("aria-live", "polite");
+assertiveSpan.setAttribute("aria-live", "assertive");
+
+politeSpan.setAttribute("role", "alert");
+assertiveSpan.setAttribute("role", "alert");
+
+politeSpan.style.cssText = styles;
+assertiveSpan.style.cssText = styles;
+
+getSingletonElementInstance("ui5-static-area").appendChild(politeSpan);
+getSingletonElementInstance("ui5-static-area").appendChild(assertiveSpan);
+
+/**
+ * Inserts the string into the respective span, depending on the mode provided.
+ *
+ * @param {string} message String to be announced by the screen reader.
+ * @param {sap.ui.core.InvisibleMessageMode} mode The mode to be inserted in the aria-live attribute.
+ */
+const announce = (message, mode) => {
+	// If no type is presented, fallback to polite announcement.
+	const span = mode === InvisibleMessageMode.Assertive ? assertiveSpan : politeSpan;
+
+	// Set textContent to empty string in order to trigger screen reader's announcement.
+	span.textContent = "";
```

Comment:
This could would then become:
`getSpan(mode).textContent = message;`
where `getSpan` is a local function for this module that returns the span, if created, or creates and then returns it.